lib/faker/default/code.rb

```diff
@@ -4,32 +4,76 @@ module Faker
   class Code < Base
     flexible :code
     class << self
-      # Generates a 10 digit NPI (National Provider Identifier
-      # issued to health care providers in the United States)
+      ##
+      # Produces a random NPI (National Provider Identifer) code.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Code.npi #=> "9804062802"
+      #
+      # @faker.version 1.9.4
       def npi
         rand(10**10).to_s.rjust(10, '0')
       end
 
-      # By default generates 10 sign isbn code in format 123456789-X
-      # You can pass 13 to generate new 13 sign code
+      ##
+      # Produces a random ISBN (International Standard Book Number) code.
+      #
+      # @param base [Number] the length of the code to generate (either 10 or 13)
```

Comment:
Shouldn't this be Integer?

```suggestion
      # @param base [Integer] the length of the code to generate (either 10 or 13)
```